staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -146,6 +146,7 @@ func NewDrainCmdOptions(f cmdutil.Factory, ioStreams genericclioptions.IOStreams
 		IOStreams:  ioStreams,
 		drainer: &drain.Helper{
 			GracePeriodSeconds: -1,
+			IgnoreErrors:       true,
```

Comment:
@pwittrock @yuzhiquan setting to true here I believe will make this new flag default.  I would first introduce the new flag to users and later in the road set it to true (if that's what you guys are looking for). Basically, the new flag is changing the current behavior for drain.

Reply:
+1

Reply:
Sure, that's my first design, i change it beacuse issue #97907 want `Follow the pattern of other kubectl subcommand, such as kubectl delete.`, but i think the review is right.

Reply:
> Sure, that's my first design, i change it beacuse issue #97907 want `Follow the pattern of other kubectl
> subcommand, such as kubectl delete.`, but i think the review is right.

Sure, I would consider this one 1.22. 
For now (as soon it get merged), I would spread the new flag in the world via blogs/docs etc.
Based on that, we would have more users aware of the flag and help move this as default behavior.

Reply:
i don't think so.  when drain multiple nodes, it should try one by one, the previous fail should not interrupt the next node.

Reply:
IMO, we don't need flag ignore-errors

Reply:
@pandaamanda Just to clarify, I am not saying we shouldn't do as default. I am just saying changing the current behavior requires to share that first to users. I would suggest add a message (like):
```
fmt.Fprintf(o.ErrOut, "DEPRECATED SOON: Aborting the drain command in a list of nodes will be deprecated. The new behavior will make the drain command go to all nodes even if one or more nodes failed during the drain. For now, users can try such experience via: --ignore-errors")
```

@pwittrock @yuzhiquan ^

Reply:
> IMO, we don't need flag ignore-errors

In our situation, ignore-errors flag is required, sometimes we need fix every error between drain nodes, specially all error may be same.It's ok to set ignore-errors as default,but that's changing current behavior, as @dougsland sayed.

Reply:
OK. SGTM